go.sum

```diff
@@ -1300,6 +1300,8 @@ github.com/hashicorp/nomad v1.10.5 h1:KGQK0Ao9L7xtO/ZagS7tXD8MvFTr5R6y7SiOGhEXzW
 github.com/hashicorp/nomad v1.10.5/go.mod h1:NElc7qdOlCrkoKaJ8jyMo4+oX1U8TIxd8NM1TKtgVYE=
 github.com/hashicorp/nomad/api v0.0.0-20250630222842-3c2a6fefd3b2 h1:oEuBbu8IP4olIERuZalsO2pvrv7Tliilvjxy129oCJM=
 github.com/hashicorp/nomad/api v0.0.0-20250630222842-3c2a6fefd3b2/go.mod h1:y4olHzVXiQolzyk6QD/gqJxQTnnchlTf/QtczFFKwOI=
+github.com/hashicorp/nomad/api v0.0.0-20251006133510-26485c45a2fb h1:QdEQr0Vq3yU4dU/zN1lVZMHEgGsHPgFRP+tAGz0B6wU=
+github.com/hashicorp/nomad/api v0.0.0-20251006133510-26485c45a2fb/go.mod h1:sldFTIgs+FsUeKU3LwVjviAIuksxD8TzDOn02MYwslE=
```

Comment:
seems odd that this is only additive. needa run `go mod tidy` which deletes the previous two lines.